src/main/java/org/testeditor/fixture/host/net/Connection.java

```
@@ -133,9 +133,9 @@ public boolean disconnect() {
         cleanup();
         boolean success = !isConnected();
         if (success) {
-            logger.info("Disconnected successfully from host : {} ", hostname);
+            logger.trace("Disconnected successfully from host : {} ", hostname);
         } else {
-            logger.info("Discoonection failed");
+            logger.error("Disconnection failed");
```

throw fixture exception instead of logging errors